templates/kubernetes/terraform/environments/dev/main.tf

```diff
@@ -19,6 +19,8 @@ module "kubernetes" {
 
   project = "<% .Name %>"
 
+  cf_signing_enabled = "<% index .Params `fileUploads` %>"
```

Comment:
i think the params stores it as a string so you may need some type casting -> bool

Reply:
Yeah, this caused the tests to fail because "yes" and "no" are not valid bool values.

Reply:
> another approach would be make a folder such as
> `kubernetes/terraform/modules/kubernetes/fileupload/main.tf` and have the current content of `backend-services.tf`
> 
> then in `kubernetes/terraform/modules/kubernetes/main.tf` have
> 
> ```
> module "fileupload" {
>   count = var.cf_signing_enabled ? 1 : 0
>   source = "./fileupload"
>   ...  some variables
> }
> ```

Oooh, I like this.

Reply:
Ha ha, easy oversight.  I would have looked at the test _eventually_. ;-)